contracts/modules/capital/SwapOperator.sol

```diff
@@ -504,6 +536,61 @@ contract SwapOperator {
     token.safeTransfer(to, amount);
   }
 
+  /**
+   * @dev Create a request for the transfer to the safe
+   * @param asset The address of the asset
+   * @param amount The amount of the asset
+   */
+  function requestAsset(address asset, uint amount) external onlySafe {
+    require(allowedAssets[asset] == true, "SwapOp: asset not allowed");
+    checkTransferAvailability(asset, amount);
+
+    transferRequest = Request(asset, amount);
+  }
+
+  /**
+   * @dev Transfer request amount of the asset to the safe
+   * @param requestedAsset The address of the asset
+   * @param requestedAmount The amount of the asset
+   */
+  function transferRequestedAsset(address requestedAsset, uint requestedAmount) external onlyController {
+    require(transferRequest.amount > 0, "SwapOp: request amount must be greater than 0");
+
+    (address asset, uint amount) = (transferRequest.asset, transferRequest.amount);
+    delete transferRequest;
+
+    require(requestedAsset == asset, "SwapOp: request assets need to match");
+    require(requestedAmount == amount, "SwapOp: request amounts need to match");
+    checkTransferAvailability(asset, amount);
+
+    _pool().transferAssetToSwapOperator(asset, amount);
+    transferAssetTo(asset, safe, amount);
+    emit TransferredToSafe(asset, amount);
+  }
+
+  /**
+   * @dev Validate that a requested asset is not sent if amount surpasses the limit
+   * @param asset The address of the asset
+   * @param amount The amount of the asset
+   */
+  function checkTransferAvailability(address asset, uint amount) internal view {
```

Comment:
If this is the fix for the reported "Lack of Outflow Limit" audit issue see the comments on notion and revisit the design decision.